_inc/lib/class-jetpack-mapbox-helper.php

```diff
@@ -39,8 +39,8 @@ public static function get_access_token() {
 
 		// If on WordPress.com, try to return the access token straight away.
 		if ( self::is_wpcom() && defined( 'WPCOM_MAPBOX_ACCESS_TOKEN' ) ) {
-			jetpack_require_lib( 'mapbox-blacklist' );
-			return wpcom_is_site_blacklisted_from_map_block( $site_id )
+			jetpack_require_lib( 'mapbox-blocklist' );
+			return wpcom_is_site_blocked_from_map_block( $site_id )
```

Comment:
This will require some changes on the WordPress.com side first, so maybe we can hold off and start there first?

```suggestion
			jetpack_require_lib( 'mapbox-blacklist' );
			return wpcom_is_site_blacklisted_from_map_block( $site_id )
```

Reply:
The wp.com side change (copying the file over to the new name) was added in r208706-wpcom.

After this diff (plus updating other changes), can delete the original.

Reply:
Perfect!